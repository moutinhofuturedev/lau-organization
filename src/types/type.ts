export type LinkCardProps = {
  url: string
  title: string
}

type UnionProps = {
  title: string
  url: string
}

export type DataProps = {
  name: string
  jobTitle: string
  links: UnionProps[]
  social: UnionProps[]
}
