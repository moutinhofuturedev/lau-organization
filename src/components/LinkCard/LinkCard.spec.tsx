import { render, screen } from "@testing-library/react";
import '@testing-library/jest-dom'
import { LinkCard } from ".";
import { LinkCardProps } from "@/types/type";

const link: LinkCardProps = {
  url: "https://assessoriavip.com.br/funnelFormLead/1d396804-ee97-4ca9-8318-7cfe1a1ed083",
  title: "Vamos conversar?"
};

describe("Component LinkCard", () => {
  it("deve renderizar link", () => {
    render(<LinkCard {...link} />);

    const titleLink = screen.getByText(link.title);

    expect(titleLink).toBeInTheDocument();
  });
})