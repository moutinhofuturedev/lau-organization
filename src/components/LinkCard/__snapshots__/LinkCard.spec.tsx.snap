// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component LinkCard deve clicar em link e abrir nova p√°gina 1`] = `[Function]`;
